docs/contributor/unsorted/architectural-overview.md

```diff
@@ -10,7 +10,7 @@ light on detail, because the precise mechanisms of various components' operation
 expected to change much faster than the general interactions between components.
 
 
-## The View From Space
+## The view From space
```

Comment:
Weird to keep `From` capitalised

Reply:
This has already been merged. That said, I agree with your comments. Those docs still have a lot of issues. In this PR I was just trying to fix the headings so the titles would show correctly in the site and in-page Nav.